scss/bootstrap/components/_editable-content.scss

```diff
@@ -2,6 +2,10 @@
   display: block;
   position: relative;
 
+  .crm-i {
+    opacity: 1;
```

Comment:
why?

Reply:
@AkA84 this was making color less visible (80%) of it's original color which should not be the case.

Reply:
"this" what? what was making the opacity different?

Reply:
Default CiviCRM style.